src/components/footer/_footer.scss

```diff
@@ -62,4 +62,16 @@
             @extend .ons-u-mb-2xs;
         }
     }
+
+    &__logo-large {
+        @include mq('2xs', 414px) {
+            display: none;
+        }
+    }
+
+    &__logo-small {
+        @include mq(415px) {
+            display: none;
+        }
+    }
```

Comment:
Both these are setting the same thing, cant they be combined? Then the include can be removed too